KarhooUISDK/CommonUI/VehicleCapacityView/NewVehicleCapacityView.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Created by Bartlomiej Sopala on 10/03/2022.
+// Copyright (c) 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import KarhooSDK
+
+public struct KHNewVehicleCapacityViewID {
+    public static let capacityView = "vehicle_capacity_view"
+    public static let passengerStack = "passenger_stack"
+    public static let baggageStack = "baggage_stack"
+    public static let baggageIcon = "baggage_image"
+    public static let baggageCapacityLabel = "baggage_capacity_label"
+    public static let capacityIcon = "passenger_capacity_image"
+    public static let passengerCapacityLabel = "passenger_capacity_label"
+}
+
+final class NewVehicleCapacityView: UIStackView {
+
+    private lazy var passengerStack = UIStackView().then {stack in
+        stack.backgroundColor = KarhooUI.colors.background1
```

Comment:
Are you sure background color works for this view? Afaik stack view instance does not have a way to customize this property. Although it could have changed.

Reply:
We both are right. Stack view has backgroundColor property, but since iOS 14. I need to fix it